core-services/release-controller/_releases/release-ocp-4.16.json

```diff
@@ -238,16 +238,16 @@
         "name": "release-openshift-ocp-installer-e2e-azure-serial-4.16"
       }
     },
-    "azure-sdn-techpreview": {
+    "azure-ovn-techpreview": {
       "optional": true,
       "prowJob": {
-        "name": "periodic-ci-openshift-release-master-ci-4.16-e2e-azure-sdn-techpreview"
+        "name": "periodic-ci-openshift-release-master-ci-4.16-e2e-azure-ovn-techpreview"
       }
     },
-    "azure-sdn-techpreview-serial": {
```

Comment:
@neisw Do you want OVN putting back into this one as well?

Reply:
Not as important here as we are using the periodic name when parsing for variants.  It would make it a little more readable but it isn't that hard to just look at the periodic name.